From 783eef588a109bb12fcc4838302619861da6452c Mon Sep 17 00:00:00 2001
From: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
Date: Tue, 12 Jun 2018 22:14:43 -0700
Subject: [PATCH 1215/1582] ASoC: uapi: sof: remove dmac id and dmac channel
 members from ipc host and dai comp def

The firmware no longer uses the dmac id and dmac channel from
topology.

Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
---
 include/uapi/sound/sof-ipc.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/uapi/sound/sof-ipc.h b/include/uapi/sound/sof-ipc.h
index d1c972a..e3c3de0 100644
--- a/include/uapi/sound/sof-ipc.h
+++ b/include/uapi/sound/sof-ipc.h
@@ -591,8 +591,6 @@ struct sof_ipc_comp_host {
 	struct sof_ipc_comp_config config;
 	enum sof_ipc_stream_direction direction;
 	uint32_t no_irq;	/* don't send periodic IRQ to host/DSP */
-	uint32_t dmac_id;
-	uint32_t dmac_chan;
 	uint32_t dmac_config; /* DMA engine specific */
 }  __attribute__((packed));
 
@@ -603,8 +601,6 @@ struct sof_ipc_comp_dai {
 	enum sof_ipc_stream_direction direction;
 	uint32_t index;
 	enum sof_ipc_dai_type type;
-	uint32_t dmac_id;
-	uint32_t dmac_chan;
 	uint32_t dmac_config; /* DMA engine specific */
 }  __attribute__((packed));
 
-- 
2.7.4

